osu.Game/Rulesets/Mods/ModHealthChallenge.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public class ModHealthChallenge : ModChallenge, IApplicableToHealthProcessor
+    {
+        public override string Name => "Health Challenge";
+        public override string Acronym => "HC";
+        public override string Description => "Hold your health or you fail";
```

Comment:
this description ought to be better; not a copywriter by any means, but maybe this is more understandable

```suggestion
        public override string Description => "Keep your health above the target!";
```